packages/thirdweb/src/wallets/in-app/native/auth/index.ts

```diff
@@ -131,9 +135,60 @@ export async function preAuthenticate(args: PreAuthArgsType) {
  * ```
  * @wallet
  */
-export async function authenticate(
-  args: AuthArgsType,
-): Promise<AuthLoginReturnType> {
+export async function authenticate(args: AuthArgsType) {
   const connector = await getInAppWalletConnector(args.client);
-  return connector.connect(args);
+  return connector.authenticate(args);
+}
+
+/**
+ * Connects a new profile (authentication method) to the current user.
+ * The connected profile can be any valid in-app wallet including email, phone, passkey, etc.
+ * The inputs mirror those used when authenticating normally.
+ *
+ * **When a profile is linked to the account, that profile can then be used to sign into the account.**
+ *
+ * This method is only available for in-app wallets.
+ *
+ * @param wallet - The wallet to link an additional profile to.
+ * @param auth - The authentications options to add the new profile.
+ * @returns A promise that resolves to the currently linked profiles when the connection is successful.
+ * @throws If the connection fails, if the profile is already linked to the account, or if the profile is already associated with another account.
+ *
+ * @example
+ * ```ts
+ * const wallet = inAppWallet();
+ *
+ * await wallet.connect({ client, strategy: "google" });
+ * const profiles = await linkProfile({ client, strategy: "discord" });
```

Comment:
Not having a wallet anywhere here feels... weird

Reply:
i actually think it fits well in the direction of 'always create an inapp wallet', no matter how you connect.

in that view you always get a 'profile' and you can always link more